tests/test_slurm.py

```diff
@@ -102,10 +102,10 @@ def test_setting_parameters(ip):
         with patch("qililab.slurm.os"):
```

Comment:
do we need any assert here? Perhaps I would do now+1 and check the assert the result